src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -43,5 +43,10 @@ internal enum ConversionKind : byte
         IntPtr,
         InterpolatedString, // a conversion from an interpolated string to IFormattable or FormattableString
         Deconstruction, // The Deconstruction conversion is not part of the language, it is an implementation detail 
+
+        // IdentityValue is not a part of the language. 
+        // It is used by lowering to ensure that trivially reduced expressions 
+        // do not become exposed to mutations if used as receivers of struct methods.
+        IdentityValue,  
```

Comment:
This conversion should be rare and should cause a pause for anyone using it (redundant use could be harmful). That is why I am not using "Identity" here, which otherwise would work just fine. So I called it "IdentityValue"
I guess "RValue" would also work

---
In reply to: [108196042](https://github.com/dotnet/roslyn/pull/18166#discussion_r108196042) [](ancestors = 108196042)